react/components/Dropdown/index.js

```diff
@@ -244,7 +244,7 @@ class Dropdown extends Component {
                 </option>
               )}
               {options.map(option => (
-                <option key={option.value} value={option.value}>
+                <option disabled={option.disabled} key={option.value} value={option.value}>
```

Comment:
Nice, @razvanudream! Can you add this to the [component's PropTypes](https://github.com/vtex/styleguide/blob/8d8dcdc9a9970f8a7f93a240c57dacef479a164f/react/components/Dropdown/index.js#L293-L296) as `PropTypes.bool`? We use PropTypes to generate the Styleguide's documentation 👇 

![image](https://user-images.githubusercontent.com/15948386/122087439-180d7880-cddb-11eb-8262-bd0d4db1de3b.png)

Also, if you can add this change to the `CHANGELOG.md`, it'd be amazing :)

Reply:
Ah, cool - just took care of those 2.  Thanks @estacioneto :)